### DISCUSSION TITLE
How to make field excluded(so that validation does not pass if that field exist)

### DISCUSSION THREAD

**Discussion by valerii15298** [go-playground/validator#1198] [Q&A]
I am using general struct, user, and do not wanna duplicate it.
I use it with gorm.
```golang
type Client struct {
	Uuid string `gorm:"primaryKey;type:uuid;default:uuid_generate_v4()"`
	FirstName string `validate:"required"`
	LastName string `validate:"required"`
}
```

Field uuid is generate by database, so in request json payload I do not wanna allow it. Can I exclude it forever so that if that field is provided the validator should return error?

There is excluded_if https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Excluded_If
But I need field to be always excluded, not by condition

**Comment by deankarn** [comment]
@valerii15298 you could use `gt=0` if only validating upon input. Or if want a different tag name use alias for encompassing.
